db/import_mb_data.py

```diff
@@ -0,0 +1,1512 @@
+import db
+from brainzutils import musicbrainz_db
+from sqlalchemy import text
+from sqlalchemy.exc import IntegrityError
+
+
+def load_artist_credit(connection, gids_in_AB):
+    artist_credit_query = text("""
+        SELECT DISTINCT artist_credit.id, artist_credit.name, artist_credit.artist_count,
+                artist_credit.ref_count, artist_credit.created
+                FROM artist_credit
+                INNER JOIN recording
+                ON artist_credit.id = recording.artist_credit
+                WHERE recording.gid in :gids OR artist_credit.id in :data
+    """)
+    MB_release_fk_artist_credit = []
+    for value in MB_release_data:
+        MB_release_fk_artist_credit.append(value[3])
+    MB_release_fk_artist_credit = list(set(MB_release_fk_artist_credit))
+
+    result = connection.execute(artist_credit_query, {'gids': tuple(gids_in_AB), 'data': tuple(MB_release_fk_artist_credit)})
+    global MB_artist_credit_data
```

Comment:
Please don't use global, return this data and then use it.

Reply:
Should fix this in every `load` function.

Reply:
Done.